library/src/main/java/com/schibsted/spain/barista/custom/ClickChildAction.java

```diff
@@ -0,0 +1,61 @@
+package com.schibsted.spain.barista.custom;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.view.View;
+import android.widget.TextView;
+
+import com.schibsted.spain.barista.util.ViewTreeAnalyzer;
+
+import org.hamcrest.Matcher;
+
+public class ClickChildAction {
+
+  public static ViewAction clickChildWithId(final int id) {
+    return new ViewAction() {
+      @Override
+      public Matcher<View> getConstraints() {
+        return null;
+      }
+
+      @Override
+      public String getDescription() {
+        return "Click on a child View with specified id";
+      }
+
+      @Override
+      public void perform(UiController uiController, View view) {
+        View child = view.findViewById(id);
+        child.performClick();
+      }
+    };
+  }
+
+  public static ViewAction clickChildWithText(final String text) {
```

Comment:
How about naming these two methods just `clickChild()`? That's what we did in Barista with `assertDisplayed()`